user_guide_src/source/outgoing/response/031.php

```diff
@@ -1,16 +1,16 @@
 <?php
 
 // Go back to the previous page.
-return redirect()->back();
+redirect()->back();
```

Comment:
It is better to put `return`, because if a dev forget it, it does not work at all.
And developers from CI3 tend to forget.

This sample file should be separated one by one?

Reply:
Okay.

Reply:
For now, I excluded those two.